### DISCUSSION TITLE
essay/2024/01/09/basecamp

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#160] [Comments]
# essay/2024/01/09/basecamp

언섹시 비즈니스에서 37signals를 다뤄줘서 재미있게 읽었습니다. 37signals는 Ruby on Rails를 만든 DHH가 창업한 회사.

https://jeho.page/essay/2024/01/09/basecamp.html

**Comment by jykim74** [comment]
여기 블로그에서 글을 읽다 보면서 생각하게 되는게 
일을 진정성과 끈기를 가지고 해야 하는구나를 생각하게 하네요.
좋은글 감사합니다..^^

**Reply by BenjaminKim** [reply]
그렇습니다. 글 내용은 달라도... 메세지는 말씀하신 게 거의 전부아닐까 싶습니다.(웃음)
